### PR TITLE
Update tailwind.config.js

### PR DESCRIPTION
**Pull request by victortokaito**
Subtitle 2: 600 -> 400 font weight

### CODE REVIEW

File: packages/web/tailwind.config.js [hunk 1]
```diff
@@ -142,13 +142,17 @@ module.exports = {
...
-      button: 700,
-      caption: 500,
+      subtitle2: 400,
+      body1: 400,
+      body2: 400,
+      button: 600,
+      caption: 400,
       overline: 400,
     },
+// body1: 500 -> 400
```
Comment (jonator) [inline packages/web/tailwind.config.js:152]: Could we remove these comments? We can see preview values in git history.

**Review by jonator — CHANGES_REQUESTED** [review]

File: packages/web/tailwind.config.js (hunk 1 above)
Comment (victortokaito) [reply packages/web/tailwind.config.js:152]: yeah lol, I'm kinda OCD about keeping notes. I'll update

**Comment by victortokaito** [comment]
@jonator am getting fails for the build--I know that's normal, but which of these are okay?

**Comment by jonator** [comment]
@victortokaito `osmosis-frontend-old` and `osmosis-refactor` are expected to fail since they used to rely on a `vercel.sh` folder to determine what branch they are on when we used to also deploy with cloudflare (`vercel.sh` is now removed).

**Pull request merged**